pyccel/codegen/python_wrapper.py

```diff
@@ -144,6 +145,12 @@ def create_shared_library(codegen,
                 verbose=verbose)
         timings['Bind C wrapping'] = time.time() - start_bind_c_compiling
         c_ast = bind_c_mod
+
+    elif language == 'cuda':
+        wrapper = CudaToCWrapper()
+        bind_c_mod = wrapper.wrap(codegen.ast)
+        c_ast = bind_c_mod
```

Comment:
Is `bind_c_mod` used later? Or can you simply do:
```suggestion
        c_ast = wrapper.wrap(codegen.ast)
```
?

Reply:
u right so far i don't need blin_c_mod : fixed 9eab7f3